### ISSUE TITLE
Some FAST components lose focus when tabIndex is changed

### ISSUE THREAD

**Issue by scomea** [microsoft/fast#4244]
This appears to be an issue related to components that delegate focus, for example fast-button and fast-text-area.  The issue is that if a component has focus and the component's tab index is changed the element loses focus and the document body becomes the activeElement.

This is problematic because components that manage tabbing and keyboard navigation will frequently change the tabIndex of focusable children.

**What are the steps to reproduce the issue?**
- navigate to https://explore.fast.design/components/fast-button
- focus on the sample button element
- inspect the element to open the debugger, navigate to "sources" and add "document" to the watch list
- the document's active element should be the fast-button at this point: 
![image](https://user-images.githubusercontent.com/7649425/104624581-9b6a4880-5648-11eb-877f-f190478b0945.png)
- type `document.activeElement.tabIndex=0` into the console
- refresh the watch panel and note the document's active element is now "body":  
![image](https://user-images.githubusercontent.com/7649425/104624970-116eaf80-5649-11eb-9506-3fe73f0930c2.png)
- repeating this process with base html elements (button, etc...) and most Fast components does not move focus except for those that delegate focus like fast-button and fast-text-area.

**What behavior did you expect?**
Changing the tabIndex of a component should not move focus

**In what environment did you see the issue?**
- Edge

**Comment by janechu** [comment]
Unfortunately `@microsoft/fast-components` has been deprecated for some time.
